projects/vdk-plugins/vdk-notebook/README.md

```diff
@@ -0,0 +1,28 @@
+# notebook
+
+A VDK plugin for working with notebooks.
```

Comment:
Maybe define the term, what is a "notebook"?

Reply:
I should have used .ipynb file instead of notebook. And I will provide a link to the VEP where it is explained more in detail.